railway/scripts/log_monitor.py

```
@@ -0,0 +1,241 @@
+        new_logs = []
+        for log in logs:
+            log_time = log.get('timestamp')
+            if log_time and log_time > self.last_log_time:
+                new_logs.append(log)
+        
+        if new_logs:
+            self.last_log_time = new_logs[-1].get('timestamp')
+        
+        return new_logs
+    
+    def monitor_logs(self, interval: int = None, single_run: bool = False):
```

Pylint (3.3.7)

[refactor] Too many branches (14/12)

(R0912)